marshmallow-metaclass.py

```diff
@@ -0,0 +1,82 @@
+from marshmallow import Schema, fields
```

Comment:
How do I import this module?  I can't do

```python
import marshmallow-metaclass
```

Reply:
I'll change the name of the module

Reply:
Hi! It is now `import mallows`. I don't know if you'll like that name